pkg/syncthing/process.go

```diff
@@ -30,7 +30,9 @@ func terminate(pid int) error {
 		return err
 	}
 
-	defer proc.Wait() // nolint: errcheck
+	if wait {
+		defer proc.Wait() // nolint: errcheck
```

Comment:
@rberrelleza this is to avoid the `wait` when finishing an `okteto up` execution. Otherwise, we are adding a 6-7s delay. I think it is safe not to wait in this case. We do wait when running `Stop(force=true)` at the beginning of the `okteto up` sequence

Reply:
I think we added this wait to prevent the case where you do down, up, and okteto tries to pick up an older pod no?

Reply:
oh wait, this is at the syncthing level, not the okteto up leve.